coderd/users.go

```diff
@@ -989,8 +989,22 @@ func (api *API) userAppearanceSettings(rw http.ResponseWriter, r *http.Request)
 		themePreference = ""
 	}
 
+	terminalFont, err := api.Database.GetUserTerminalFont(ctx, user.ID)
+	if err != nil {
+		if !errors.Is(err, sql.ErrNoRows) {
+			httpapi.Write(ctx, rw, http.StatusInternalServerError, codersdk.Response{
+				Message: "Error reading user settings.",
+				Detail:  err.Error(),
+			})
+			return
+		}
+
+		terminalFont = ""
```

Comment:
I didn't look at the FE code, but I suppose it handles empty string / default font?

Reply:
Yes 👍